### PR TITLE
Add logo for docsstring

### LINKED ISSUE #482: A new icon for docstrings in libraries

### PR DESCRIPTION
**Pull request by tbille**
## Done

- Add logo for docs string
- Add logo for update date

## QA

- Check out this feature branch
- Run the site using the command `dotrun`
- http://0.0.0.0:8045/activemq/libraries/charms.apache.v2.http
- Make sure it matches the design: https://zpl.io/2pEQzWE

image

## Issue / Card

Fixes #482

### CODE REVIEW

File: templates/details/libraries/libraries_layout.html [hunk 1]
```diff
@@ -63,20 +65,10 @@ <h3>{{ docstrings["name"] }}</h3>
                 <a href="#" class="is-always-color-link has-icon"><i class="p-icon--download">Download</i><span>Download</span></a>
               </li>
               <li class="p-inline-list__item has-icon">
-                <i class="p-icon--revision">Last revision</i>26 June 2019
+                <i class="p-icon--update">Update</i>26 June 2019
```
Comment (bartaz) [inline templates/details/libraries/libraries_layout.html:68]: Stupid question, should it be "Update", or "Updated" or "Last updated" or similar?

Comment (tbille) [reply templates/details/libraries/libraries_layout.html:68]: Not a stupid question. `Last updated` is good.
Fixing it now

Comment (tbille) [reply templates/details/libraries/libraries_layout.html]: Fixed

**Review by sowasred2012 — APPROVED** [review]
LGTM

**Pull request merged**